pillar/nginx/odlvideo.sls

```diff
@@ -37,8 +37,7 @@ nginx:
           config:
             - server:
                 - server_name:
-                    - {{ server_domain_name }}
-                    - techtv.mit.edu
+                    {{ server_domain_name }}
```

Comment:
Pluralize this as well. Also, since it is now a list it should be moved up a line
```
config:
  - server:
    - server_name: {{ server_domain_names }}
```